test/interface_c.cpp

```diff
@@ -0,0 +1,182 @@
+#include <gtest/gtest.h>
+
+extern "C" {
+    void store_function_pointers(int num_fptrs, const char * fptr_names[], void * fptrs[]);
+    int trixi_version_library_major();
+    int trixi_version_library_minor();
+    int trixi_version_library_patch();
+    const char* trixi_version_library();
+    const char* trixi_version_julia();
+    const char* trixi_version_julia_extended();
+    void trixi_initialize(const char * project_directory, const char * depot_path);
+    int trixi_initialize_simulation(const char * libelixir);
+    void trixi_finalize_simulation(int handle);
+    void trixi_finalize();
+    int trixi_is_finished(int handle);
+    void trixi_step(int handle);
+    double trixi_calculate_dt(int handle);
+    int trixi_ndims(int handle);
+    int trixi_nelements(int handle);
+    int trixi_nvariables(int handle);
+    void trixi_load_cell_averages(double * data, int handle);
+    void trixi_eval_julia(const char * code);
```

Comment:
```suggestion
#include path/to/trixi.h
```
? (otherwise, you will always need to adapt this as well when adding/modifying functions)

Reply:
Ok!
This looks a little strange to me, but it works!